src/common/src/session_config/mod.rs

```diff
@@ -338,6 +341,9 @@ pub struct ConfigMap {
     /// optimizations. Defaults to false.
     /// Setting this to true will always set `ENABLE_TWO_PHASE_AGG` to false.
     force_two_phase_agg: ForceTwoPhaseAgg,
+
+    /// Enable share plan to construct a DAG plan.
```

Comment:
```suggestion
    /// Enable sharing of common sub-plans.
    /// This means that DAG structured query plans can be constructed,
    /// rather than only tree structured query plans.
```